src/pages/ProductDetailsPage/ProductDetailsPage.css

```diff
@@ -0,0 +1,3 @@
+
+
```

Comment:
i think you can remove this file?

Reply:
yes will remove it, btw thanks for reviewing code in detail your review are so helpful 🙌